internal/util/vault.go

```diff
@@ -58,6 +58,8 @@ Example JSON structure in the KMS config is,
 		"vaultNamespace": "",
 		"vaultPassphraseRoot": "/v1/secret",
 		"vaultPassphrasePath": "",
+                "vaultTokenAuth": true,
```

Comment:
the indentation is not proper here.

Reply:
ah.. will correct it.

Reply:
This is not fixed yet, please don't mark it resolved it if it's not fixed.